.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+      - id: check-added-large-files
+
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.26.3
+    hooks:
+      - id: yamllint
+        args: [-c=.yamllint]
+
+  - repo: https://github.com/ansible-community/ansible-lint.git
+    rev: v6.10.2
+    hooks:
+      - id: ansible-lint
+        pass_filenames: false
+        entry: ansible-lint --force-color -v
```

Comment:
Not sure if these are required for you specifically. I originally had some issues in my project which runs inside a VSCode Devcontainer which I added these params for, but probably won't be necessary for you. See how you go with dropping these from your config :)

Reply:
thanks, trying ...